criu/image.c

```diff
@@ -580,8 +580,10 @@ int open_image_dir(char *dir)
 	}
 
 	ret = install_service_fd(IMG_FD_OFF, fd);
-	if (ret < 0)
+	if (ret < 0) {
+		pr_err("install_service_fd failed.\n");
```

Comment:
Left this as `install_service_fd failed` since other errors in this file were also generically written. `install_service_fd` also has its own error messaging, so this message should be fine.

Reply:
revert and add logging in https://github.com/checkpoint-restore/criu/blob/v3.14/criu/crtools.c#L155 something like `"Couldn't open image dir: %s", opts.imgs_dir`